src/me/sheimi/sgit/dialogs/CloneDialog.java

```diff
@@ -64,6 +64,20 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
             mLocalPath.setText(Repo.TEST_LOCAL);
         }
 
+        mRemoteURL.setOnFocusChangeListener(new View.OnFocusChangeListener() {
+            @Override
+            public void onFocusChange(View view, boolean hasFocus) {
+                if (!hasFocus) {
+                    final String remoteUrl = mRemoteURL.getText().toString();
+                    final int lastSlash = remoteUrl.lastIndexOf("/");
+                    final String localHint = remoteUrl.substring(lastSlash + 1);
```

Comment:
I agree.  Even though git handles this situation itself, it is certainly more visually appealing if it is never there to begin with.